packages/image_picker/image_picker/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -644,6 +762,15 @@
 			defaultConfigurationIsVisible = 0;
 			defaultConfigurationName = Release;
 		};
+		6801C83F2555D726009DAF8D /* Build configuration list for PBXNativeTarget "RunnerUITests" */ = {
```

Comment:
This the hook for setting build configuration settings for the new test target (like a base xcconfig file)